webpack.procure.config.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
```

Comment:
As I see, you created a copy of `webpack.config.js` and fixed some paths. I think, it will be better to create a basic config and then inherit these two from it.